### DISCUSSION TITLE
What's the status of Intl API on React Native Windows with Hermes enabled?

### DISCUSSION THREAD

**Discussion by juhasuni** [microsoft/react-native-windows#13000] [General]
The official Hermes has had Intl API (partially implemented) for some time now. It appears Microsoft manages a fork of the Facebook's package where the version number is not aligned with the official Hermes version.

But running latest React Native Windows (0.73.x) Hermes enabled doesn't provide Intl API. Not even when bumping the Nuget package of `Microsoft.JavaScript.Hermes` to `0.1.20`.

We've been polyfilling Intl with an ancient Node.js polyfill and it's far from ideal.

```js
console.log(typeof Intl); // --> undefined
```

**Comment by tero-paananen** [comment]
Hermes 0.1.21 update is still missing Intl https://github.com/microsoft/react-native-windows/commit/a1b2139eec4ce5416a6098b737ea2662af3d896f

**Reply by anramanath** [reply]
Seems to be available in 0.1.27+

**Reply by juhasuni** [reply]
Well that's great news! We'll have to try this out.
